tools/wpt/browser.py

```diff
@@ -47,6 +47,11 @@ class Browser(object):
     def __init__(self, logger):
         self.logger = logger
 
+    @abstractmethod
```

Comment:
I would really wish splitting this out into a separate PR, but I won't block you on this.